.github/scripts/generate-plugin.sh

```diff
@@ -4,7 +4,7 @@ set -Eeuo pipefail
 trap 'echo Error encountered while executing the script.' ERR
 
 SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
-ROOT_REPOSITORY_DIR="${SCRIPT_DIR}/../.."
+ROOT_REPOSITORY_DIR="$(realpath "${SCRIPT_DIR}/../..")"
```

Comment:
Just a minor adjustment in order to normalize path.

It's better to have `foo/bar` displayed instead of `foo/bar/../bar` even if both are equivalent.